### PR TITLE
feat: Action Webhooks UI changes

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

[This PR](https://github.com/PostHog/posthog/pull/20431) wad denied 😢 pulled out just the UI parts

## Changes

* Prettify the Actions editing UI a bit
* Fixes the typing for api pagination

|Before|After|
|----|----|
| Screenshot 2024-02-20 at 10 35 38 | Screenshot 2024-02-20 at 10 35 55 |

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/lib/constants.tsx [hunk 1]
```diff
@@ -207,6 +207,7 @@ export const FEATURE_FLAGS = {
     SESSION_REPLAY_DOCTOR: 'session-replay-doctor', // owner: #team-replay
     REPLAY_SIMILAR_RECORDINGS: 'session-replay-similar-recordings', // owner: #team-replay
     SAVED_NOT_PINNED: 'saved-not-pinned', // owner: #team-replay
+    MULTIPLE_ACTION_WEBHOOKS: 'multiple-action-webhooks', // owner: @benjackwhite
```
Comment (mariusandra) [inline frontend/src/lib/constants.tsx:210]: Is this used anywhere?

File: frontend/src/scenes/actions/webhooks/actionWebhooksLogic.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import { lemonToast } from '@posthog/lemon-ui'
+import { actions, afterMount, kea, key, listeners, path, props, reducers } from 'kea'
+import { forms } from 'kea-forms'
+import { loaders } from 'kea-loaders'
+import { beforeUnload } from 'kea-router'
+import api from 'lib/api'
+import { toParams } from 'lib/utils'
+
+import { ActionType, HookConfigType } from '~/types'
+
```
Comment (mariusandra) [inline frontend/src/scenes/actions/webhooks/actionWebhooksLogic.ts:10]: `Module '"~/types"' has no exported member 'HookConfigType'.`

Also this logic seems to be unreferenced from anywhere else. 🤔

**Review by mariusandra — COMMENTED** [review]
Definitely looks nicer, but there's a TS error (`types.ts`) that's easiest to solve by removing `actionWebhooksLogic`.

File: frontend/src/lib/constants.tsx (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/lib/constants.tsx:210]: Ah whoops. My git picking skills are not good....

File: frontend/src/scenes/actions/webhooks/actionWebhooksLogic.ts (hunk 2 above)
Comment (benjackwhite) [reply frontend/src/scenes/actions/webhooks/actionWebhooksLogic.ts]: Yeah another bad git pick

**Review by mariusandra — APPROVED** [review]
🎉

**Pull request merged**